test/cpp/tensorexpr/tests_setup.py

```diff
@@ -0,0 +1,88 @@
+import sys
```

Comment:
what is this magic file?

Reply:
It was also copied, we probably don't need it :)